From 8760102a2e73747c6ae570c699c3da6752c9efb7 Mon Sep 17 00:00:00 2001
Message-Id: <8760102a2e73747c6ae570c699c3da6752c9efb7@dist-git>
From: Jiri Denemark <jdenemar@redhat.com>
Date: Tue, 26 Feb 2019 21:02:49 +0100
Subject: [PATCH] cpu_map: Add more signatures for Skylake-Client CPU models
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Jiri Denemark <jdenemar@redhat.com>
Reviewed-by: Ján Tomko <jtomko@redhat.com>
(cherry picked from commit 367d96a5d6b04bf25d025ed59a7079d71f843c56)

https://bugzilla.redhat.com/show_bug.cgi?id=1558558
https://bugzilla.redhat.com/show_bug.cgi?id=1687515

Conflicts:
	src/cpu_map/x86_Skylake-Client-IBRS.xml
	src/cpu_map/x86_Skylake-Client.xml
            - cpu_map split not backported

Signed-off-by: Jiri Denemark <jdenemar@redhat.com>
Reviewed-by: Ján Tomko <jtomko@redhat.com>
---
 src/cpu/cpu_map.xml | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/src/cpu/cpu_map.xml b/src/cpu/cpu_map.xml
index 095ea0a839..1382c29cd1 100644
--- a/src/cpu/cpu_map.xml
+++ b/src/cpu/cpu_map.xml
@@ -1680,6 +1680,11 @@
 
     <model name='Skylake-Client'>
       <signature family='6' model='94'/> <!-- 0506e0 -->
+      <signature family='6' model='78'/> <!-- 0406e0 -->
+      <!-- These are Kaby Lake and Coffee Lake successors to Skylake,
+           but we don't have specific models for them. -->
+      <signature family='6' model='142'/> <!-- 0806e0 -->
+      <signature family='6' model='158'/> <!-- 0906e0 -->
       <vendor name='Intel'/>
       <feature name='3dnowprefetch'/>
       <feature name='abm'/>
@@ -1748,6 +1753,11 @@
 
     <model name='Skylake-Client-IBRS'>
       <signature family='6' model='94'/> <!-- 0506e0 -->
+      <signature family='6' model='78'/> <!-- 0406e0 -->
+      <!-- These are Kaby Lake and Coffee Lake successors to Skylake,
+           but we don't have specific models for them. -->
+      <signature family='6' model='142'/> <!-- 0806e0 -->
+      <signature family='6' model='158'/> <!-- 0906e0 -->
       <vendor name='Intel'/>
       <feature name='3dnowprefetch'/>
       <feature name='abm'/>
-- 
2.21.0

